### PR TITLE
Add new flash card

### PR DESCRIPTION
**Pull request by rsmidt**
It's now possible to create a flashcard. I've built a simple bottom navigation using React Router and Material UI.

### CODE REVIEW

File: src/graphql/fragments.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { gql } from "@apollo/client";
+
+export const COMMON_FLASHCARD_FIELDS = gql`
+  fragment CommonFlashcardFields on FlashcardModel {
```
Comment (ThorbenSchiller) [inline src/graphql/fragments.ts:4]: Wollen wir es mit dem Suffix "Fragment" benennen?
Analog zu -Query und -Mutation?

File: src/vite-env.d.ts [hunk 2]
```diff
@@ -1,2 +1,71 @@
 /// <reference types="vite/client" />
 /// <reference types="@emotion/react/types/css-prop" />
+
+export type SpacingValue = number | string;
```
Comment (ThorbenSchiller) [inline src/vite-env.d.ts:4]: Brauchen wir die Definitionen noch?

File: src/pages/CreateFlashcardPage/CreateFlashcardPage.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+  return (
+    <Container>
+      <Snackbar
+        open={showSuccessMessage}
+        onClose={() => setShowSuccessMessage(false)}
+        autoHideDuration={4000}
+        message="Flashcard created successfully."
+      />
+      <Typography variant="h1">New Flashcard</Typography>
```
Comment (ThorbenSchiller) [inline src/pages/CreateFlashcardPage/CreateFlashcardPage.tsx:53]: `variant="h1"` is pretty "big" - maybe we use something smaller?
Like `<Typography variant="h3" component="h1">`

Comment (rsmidt) [reply src/pages/CreateFlashcardPage/CreateFlashcardPage.tsx:53]: I've used the [responsive font size helper](https://next.material-ui.com/customization/typography/#heading-responsive-font-sizes) from MUI so on mobile it doesn't appear that big to me. Did you check it out on a small viewport?

File: src/graphql/fragments.ts (hunk 1 above)
Comment (rsmidt) [reply src/graphql/fragments.ts:4]: So instead of `CommonFlashcardFields` it would be `CommonFlashcardFragment`?

Comment (ThorbenSchiller) [reply src/graphql/fragments.ts:4]: Yes :)

File: src/pages/CreateFlashcardPage/CreateFlashcardPage.tsx (hunk 3 above)
Comment (ThorbenSchiller) [reply src/pages/CreateFlashcardPage/CreateFlashcardPage.tsx:53]: No, I only checked the desktop viewport.
But then let's keep it like that as the main focus is on mobile for now

**Review by ThorbenSchiller — APPROVED** [review]

**Pull request merged**